notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/PushChannelEnricher.java

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+public class PushChannelEnricher implements InstallationEnricher {
+    private String channel;

```

Comment:
Consistency in naming for member variables

Reply:
Fixed in 583cdf30d92d7f9a3962e8b37fae21d9037b4511